assets/css/pages-frontend.scss

```diff
@@ -0,0 +1,9 @@
+/**
+ * Course Completed Page
+ */
+.course-completed .entry-title {
```

Comment:
Just looking at Twenty Twenty-One, should we hide the `.entry-header`? Because it has some border, padding, and margin too.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/4273/commits/b458d37627ebec5f13ce47acc779d7e647b95cc2.